go.mod

```diff
@@ -32,13 +31,13 @@ require (
 	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110 // indirect
 	golang.org/x/text v0.3.5 // indirect
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.20.4
+	k8s.io/api v0.21.0-rc.0
```

Comment:
Just a pointer - this seems risky, but apparently newest API does exactly that too.

https://github.com/openshift/api/blob/47be53705a1362bfe78926a87dc5d12e0480accf/go.mod#L10-L12